instHWWWW/instHWWWW/Views/RecentlyScreen/TodayTableViewCell.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  TodayTableViewCell.swift
+//  instHWWWW
+//
+//  Created by Александр Троицкий on 21.10.2022.
+//
+
+import UIKit
+
+/// Настройка ячейки "сегодня"
+final class TodayTableViewCell: UITableViewCell {
+    
+    // MARK: - IBOutlet
+    @IBOutlet private var avatarPersonImageView: UIImageView!
+    
+    @IBOutlet private var postImageView: UIImageView!
+    
+    @IBOutlet private var subscribeButton: UIButton!
+    
+    @IBOutlet private var likeLabel: UILabel!
+    
+    // MARK: - Lifecycle
+    override func awakeFromNib() {
+        super.awakeFromNib()
+    }
+
+    override func setSelected(_ selected: Bool, animated: Bool) {
+        super.setSelected(selected, animated: animated)
+    }
+    
+    // MARK: - Public Methods
+    
+    func refresh(_ model: Recently) {
+         avatarPersonImageView.image = UIImage(named: model.profileImageName)
+
+        likeLabel.attributedText = NSMutableAttributedString()
+            .bold("\(model.profileImageName) ")
+            .normal("\(model.commentText) ")
+            .grayTextColor(" \(model.timeText)")
+        postImageView.image = UIImage(named: model.imageName)
```

Comment:
```suggestion
        postImageView.image = UIImage(named:.imageName)
```
забываем про слово model